api/server/router/network/backend.go

```diff
@@ -0,0 +1,22 @@
+package network
+
+import (
+	// network config needs to be pull out to here
+	"github.com/docker/docker/daemon/network"
+
+	"github.com/docker/libnetwork"
+)
+
+// Backend is all the methods that need to be implemented to provide
```

Comment:
lol, I missed that line above. Hopefully it looks better now? I'm not sure that comment is terribly content filled, but I gave it another shot.